content/posts/2019-08-03---Going-from-ember-to-vue.md

```diff
@@ -146,14 +156,17 @@ I just wanted some examples of the major key pieces.
 First off, all of the markup is wrapped in a `template`.
 After that, you are writing normal HTML.
 You call other components just like `<AngleBracket />` components in Ember, but the differences (for now) is that you have to import them.
-This saves from potential name collision, as an npm package might have a component with the same name as one of yours.
-You could do `import SomeCustomComponent as Monkey from '@something/components';` and rename a component without any magic.
+This saves you from potential name collision, as an npm package might have a component with the same name yours.
+You could do `import SomeCustomComponent as Monkey from '@something/components';` and rename a component easily.
+Again, this is something in an RFC for Ember already but it cements a point I made in an earlier post that [the resolver is a bit too clever sometimes](/posts/a-few-thoughts-on-ember/#3-make-the-resolver-less-clever).
 
 ## Methods/Actions
 
-When calling a component, we have similar patterns as Ember.
-You need to append a `:` to properties you pass down to a component and `@` for events that map to functions (think `actions`).
+When using a component, we have similar patterns as Ember.
+You need to append a `:` to properties you pass down to a component.
+So, if my parent component has a property called `author` and I want to pass `author.name` down to a child component as `fullname`, it would look like this: `<AuthorInfo :fullname="author.name" />`.
 `methods` are the equivalent of `actions` in Ember -- if there is a function that is going to be called from the template, it is in the `methods` hash.
+Functions can live outside of that hash (obviously) but this tells you what you expect to interact with your markup.
```

Comment:
this is handy -- ember on the other hand has moved away from this, because people just want to use functions on the class -- it's super nice -- only mishap you can run in to is when you accidentally name an action the same an a component API -- with Ember Components, that's a lot more likely than in Octane's Glimmer Components (Ember has like.. 30+ apis, and Glimmer has 4)